Warehouse.Persistence.PostgreSQL/Configuration/Contstants/DapperBasketConstants/DapperBasketReadConst.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Warehouse.Persistence.PostgreSQL.Configuration.Contstants.DapperBasketConstants;
+public static class DapperBasketReadConst
```

Comment:
I want to divide them by Mutated state and read state to follow the approach of CQRS:
MutatableQueryContant.cs  -> Create, Update, Delete -> MutatableQueryContant.MutateProductConstant
ReabableQueryConstant.cs  -> Read  -> > ReadableQueryContant.ReadProductConstant
or folder 
Constants/MutatableQueries/MutatableQueryProductConstant.cs
Constants/ReadableQueries/ReadbleQueryProductConstant.cs

Or something like that

Reply:
![image](https://github.com/georgiev84/DigitallWarehouse_v3/assets/24383099/e996c251-5114-4fe3-8011-648b261fb4da)
I prefer this one